src/board.tsx

```diff
@@ -106,108 +104,139 @@ export const Board = ({
   const maxColumnOffset = Math.floor((columns.length ?? 0) - width / 36 + 1);
   const allColumnsVisible = (columns.length ?? 0) <= width / 36;
 
-  const checkAndSetTop = useCallback(
-    (newIndex: number) => {
-      if (newIndex * 7 - top + 10 > height - 7) {
-        setTop((prev) => prev + 7);
-      }
+  const checkAndSetOffsets = useCallback(
+    (dimension: "top" | "left", offset: number) => {
+      const { top, left } = scrollOffset;
 
-      if (newIndex * 7 - top - 10 < 0) {
-        setTop((prev) => Math.max(prev - 7, 0));
-      }
-    },
-    [top, height],
-  );
+      if (dimension === "top") {
+        if (offset * 7 - top + 10 > height - 7) {
+          setScrollOffset((prev) => ({ ...prev, top: prev.top + 7 }));
+        }
 
-  const checkAndSetLeft = useCallback(
-    (newIndex: number) => {
-      if (newIndex * 36 - left + 36 > width) {
-        setLeft((prev) => prev + 36);
-      }
+        if (offset * 7 - top - 10 < 0) {
+          setScrollOffset((prev) => ({
+            ...prev,
+            top: Math.max(prev.top - 7, 0),
+          }));
+        }
+      } else {
+        if (offset * 36 - left + 36 > width) {
+          setScrollOffset((prev) => ({ ...prev, left: prev.left + 36 }));
+        }
 
-      if (newIndex * 36 - left - 36 < 0) {
-        setLeft((prev) => Math.max(prev - 36, 0));
+        if (offset * 36 - left - 36 < 0) {
+          setScrollOffset((prev) => ({
+            ...prev,
+            left: Math.max(prev.left - 36, 0),
+          }));
+        }
       }
     },
-    [left, width],
+    [scrollOffset, height, width, setScrollOffset],
   );
 
   useInput((input, key) => {
+    if (inputDisabled) return;
+
+    if (input === "p") {
+      openModal("updatePriority");
+      return;
+    }
+    if (input === "a") {
+      openModal("updateAssignee");
+      return;
+    }
+    if (input === "m") {
+      openModal("moveIssue");
+      return;
+    }
```

Comment:
These aren't documented as keybinds on the board view (I thought this wasn't supported yet when I was using this, but to my surprise it's already here, just invisible)